src/main/java/Event.java

```diff
@@ -0,0 +1,77 @@
+public class Event extends Task{
+
+    private final String from_time;
+    private final String to_time;
```

Comment:
Try to store the `from_time` and `to_time` with LocalDate, this will be done in Level-8.